.github/workflows/image-scan.yml

```
@@ -0,0 +1,22 @@
+    uses: truefoundry/workflows/.github/workflows/update-grype-report.yml@add-vul-report
+    with:
+      grype_config_file: resolver-grype-config.yaml
+      image_artifact_name: 'elasti-resolver'
+      artifactory_repository_url: ${{ vars.TRUEFOUNDRY_ARTIFACTORY_PUBLIC_REPOSITORY }}
+  image-scan-build-operator:
+    name: Image Vulnerability Scan for Build Operator
+    uses: truefoundry/workflows/.github/workflows/update-grype-report.yml@add-vul-report
+    with:
+      grype_config_file: operator-grype-config.yaml
+      image_artifact_name: 'elasti-operator'
+      artifactory_repository_url: ${{ vars.TRUEFOUNDRY_ARTIFACTORY_PUBLIC_REPOSITORY }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)